app/src/main/java/de/stephanlindauer/criticalmaps/helper/clientinfo/DeviceInformation.java

```diff
@@ -26,12 +26,6 @@ public static String getString() {
         returnString += "HARDWARE= " + Build.HARDWARE + "\n";
         returnString += "ID= " + Build.ID + "\n";
         returnString += "MANUFACTURER= " + Build.MANUFACTURER + "\n";
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
```

Comment:
Permission is only required starting in O, hence the check. That aside I'm perfectly fine with removing it altogether since I don't think the serial gives us any valuable insight.

Reply:
Actually it seems it indeed needs to be removed as it is already removed in the develop branch. See: https://github.com/criticalmaps/criticalmaps-android/commit/99b97a555c2444357374863238769a0667553575